server/commands/connected_objects.go

```diff
@@ -21,10 +21,58 @@ type ConnectedObjectsRequest struct {
 	StoreID          string
 	ObjectType       string
 	Relation         string
-	User             *openfgapb.ObjectRelation
+	User             isUser_Ref
 	ContextualTuples []*openfgapb.TupleKey
 }
 
+type isUser_Ref interface {
```

Comment:
I introduced this interface to distinguish between different user types. This structure resembles what would be generated from the protobuf compiler if our protobuf API distinguished between objects, typed wildcards, and object#relation users.

I like it in theory, but I'm not sure how I feel about it in terms of the code changes that were required herein.

During review, if you have alternative suggestions that you believe would make this handling more elegant, I would love to hear that feedback!